llvm/test/CodeGen/RISCV/corev/bi.ll

```diff
@@ -1,18 +1,24 @@
-; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
-; RUN: llc -O0 -mtriple=riscv32 -mattr=+xcvbi -verify-machineinstrs < %s \
-; RUN:   | FileCheck %s
+; RUN: llc -O0 -mtriple=riscv32 -mattr=+xcvbi -verify-machineinstrs -o %T/non-optimized.s < %s
+; RUN: FileCheck %s --check-prefixes=CHECK,CHECK_NOPT -input-file=%T/non-optimized.s
+; RUN: llc -O3 -mtriple=riscv32 -mattr=+xcvbi -verify-machineinstrs -o %T/optimized.s < %s
+; RUN: FileCheck %s --check-prefixes=CHECK,CHECK_OPT -input-file=%T/optimized.s
```

Comment:
You can still use `update_llc_test_checks.py` if you give each FileCheck command a different prefix, e.g. if line 2 uses `--check-prefix=CHECK-NOOPT`, and line 4 is `--check-prefix=CHECK-OPT`

It may be that python script only works when piping the output from `llc` to `FileCheck` rather than using  `-input-file`... but I'm not 100% sure on that